packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -41,7 +41,7 @@ type UploadParams = BaseUploadParams & {
 }
 
 type StreamUploadParams = BaseUploadParams & {
-  stream: ReadStream
+  stream: ReadStream | NodeJS.ReadableStream | ReadableStream<Uint8Array> | null
```

Comment:
Is this truly `null` or is it optional like `stream?: ReadStream | NodeJS.ReadableStream | ReadableStream<Uint8Array>`